### DISCUSSION TITLE
Can Zeebe Client support .Net 4.8

### DISCUSSION THREAD

**Discussion by Thilo1** [camunda-community-hub/zeebe-client-csharp#651] [General]
Good Evening,

is it possible to provide the whole or a part of the zeebee client nuget package to .Net 4.8 projects ?
Background is this: I have a project that runs on .net framework 4.8, but i need to start deployed camunda instances.
What is in your opinion the way to go?

Best Regards
Thilo

**Comment by ChrisKujawa** [comment]
Sorry, there is no plan from my side to support .net framework. Especially since I'm a Linux user and developing against the framework afaik will not work. Furthermore, it see no real value in it.

Feel free to fork the project and adjust it to your needs.

**Reply by Thilo1** [reply]
Thanks for the quick reply. It's ok, i found a way that allows me to use .net core 8 for it. So from my side the discussion is solved satisfactorily.
